VehicleCode/Output/Dashboard.cpp

```diff
@@ -0,0 +1,90 @@
+#include "Dashboard.h"
+#include "Actuator.h"
+#include "Screen.h"
+#include "LEDArray.h"
+#include "Servo.h"
+#include "Horn.h"
+#include "../SensorID.h"
+
+// #include <Arduino.h>
+// #include <FlexCAN_T4.h>
+
+// Global variables
+#define BAUDRATE 250000
+
+// CHECK: define function
+// Constructor
+Dashboard::Dashboard(unsigned long startTime) {
+    // load parameters
+    this->startTime = startTime;
+
+    // Initialize CAN bus
+    FlexCAN_T4<CAN2, RX_SIZE_256, TX_SIZE_16> can2;
+    can2.begin();
+    can2.setBaudRate(BAUDRATE);
+
+    // Instantiate the actuators
+    Screen* lapScreen = new Screen();
```

Comment:
I think these objects should be initialized in platformio project that will be put on the teensy that has a dashboard object. An array of Actuators* can then be passed into the constructor of the dashboard class
```suggestion
    // TODO include pin information
```